api/authenticator.py

```diff
@@ -784,10 +784,18 @@ class AuthenticationProvider(object):
     LIBRARY_SETTINGS = [
         { "key": EXTERNAL_TYPE_REGULAR_EXPRESSION,
           "label": _("External Type Regular Expression"),
+          "description": _("Derive a patron's external type from their identifier."),
```

Comment:
Maybe this should explain what an external type is. Since the setting to map external types to lanes isn't in the admin interface yet there's nowhere else you would see it.